packages/lodestar/package.json

```diff
@@ -12,7 +12,25 @@
     "url": "https://github.com/ChainSafe/lodestar/issues"
   },
   "version": "0.37.0",
-  "main": "lib/index.js",
+  "type": "module",
+  "exports": {
+    ".": {
+      "import": "./lib/index.js"
+    },
+    "./chain": {
+      "import": "./lib/chain/index.js"
```

Comment:
Why is chain exported? How is using it directly?

Reply:
tests

Reply:
So requiring from src soesn't work? It doesnt feel super clean to have an extra export for something that not used outside of tests

Reply:
relative path imports work. I'll remove this